### PR TITLE
Update to use new Fabric v1 (beta/alpha2) struct changes

### PR DESCRIPTION
**Pull request by rmohta**
Ability to read data using Fabric v1.0

> NOTE: For the chain code to work, will need updated version of hyperledger/fabric-ccenv

### CODE REVIEW

**Comment by dshuffma-ibm** [comment]
thanks, i'll have to hold off on accepting this to coordinate this with the bluemix service.  then again maybe I should just make a marbles v4 with this change...

**Comment by rmohta** [comment]
@dshuffma-ibm Cool. Would you be open to having this open so that I can send in all the changes - code, nodejs and documentation - related to running this on non-bluemix platform?

**Comment by dshuffma-ibm** [comment]
yep, that will be fine @rmohta

**Comment by gemtot** [comment]
Can we expand this to include the Node SDK changes required to enable the demo to work with 1.0.0-alpha2?

File: chaincode/src/marbles/read_ledger.go [hunk 1]
```diff
@@ -175,19 +177,19 @@ func getHistory(stub shim.ChaincodeStubInterface, args []string) pb.Response {
...
+                historyData, err := resultsIterator.Next()
 		if err != nil {
 			return shim.Error(err.Error())
 		}
 
 		var tx AuditHistory
-		tx.TxId = txID                             //copy transaction id over
-		json.Unmarshal(historicValue, &marble)     //un stringify it aka JSON.parse()
-		if historicValue == nil {                  //marble has been deleted
+		tx.TxId = historyData.TxID                             //copy transaction id over
```
Comment (mrshah-at-work) [inline chaincode/src/marbles/read_ledger.go:186]: For alpha2, It should be `TxId` instead of `TxID`

**Pull request merged**